### PR TITLE
add bulk receive view

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

First tasks for bulk send 

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

### CODE REVIEW

**Comment by owocki** [comment]
yay [i got this working today!!!](http://bits.owocki.com/190c055a2202/Screen%20Shot%202018-11-17%20at%206.58.09%20PM.png)

**Comment by owocki** [comment]
since this is super greenfield, im going to deploy unilaterally so we can test in prod

File: app/retail/templates/shared/grants_nav.html [hunk 1]
```diff
@@ -41,14 +41,11 @@
       {% endif %}
 
       {% if not user.is_authenticated %}
-        <a href="{% url 'social:begin' 'github' %}?next=/" class="nav-link interior">
```
[the remark this answers is not in the document]
Comment (octavioamu) [reply app/retail/templates/shared/grants_nav.html]: Mmm I think not, this was and old PR so the actual one should be more updated than this one.

**Comment by octavioamu** [comment]
NICE!

**Pull request merged**